Source/WebInspectorUI/UserInterface/Views/ScreenshotsTimelineView.css

```diff
@@ -27,6 +27,7 @@
     white-space: nowrap; 
     overflow-x: auto;
     scroll-snap-type: x mandatory;
+    background-color: hsl(0, 0%, 90%);
```

Comment:
Nit: Can we share a variable here and for `.content-view.graphics-overview` and just have the variable change based on light/dark mode?

Reply:
i think usually we normally create shared code once we hit three separate usages

(also i cant think of a good name for this that isn't already being used 😅)